api4/cloud.go

```diff
@@ -67,6 +68,31 @@ func getSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write(json)
 }
 
+func getSubscriptionStats(c *Context, w http.ResponseWriter, r *http.Request) {
+	subscription, appErr := c.App.Cloud().GetSubscription()
```

Comment:
We also need to make sure they are on a cloud license before we try to get the subscription something like: 

```
if c.App.Srv().License() == nil || !*c.App.Srv().License().Features.Cloud {
		c.Err = model.NewAppError("Api4.getSubscriptionStats", "api.cloud.license_error", nil, "", http.StatusNotImplemented)
		return
	}
```